guide/.vuepress/sidebar.ts

```diff
@@ -159,11 +159,6 @@ export default {
 			text: 'Voice',
 			children: [
```

Comment:
This entire object should be removed, since there's a nav bar button for the voice pages now

Reply:
Ok.